tests/Oryx.Integration.Tests/Php/PhpFpmConfigTest.cs

```diff
@@ -73,6 +73,59 @@ public PhpFpmConfigTest(ITestOutputHelper output, TestTempDirTestFixture fixture
                 failureOutputText);
         }
 
+        [Fact]
```

Comment:
```suggestion
        [Fact, Trait("category", "php-8.0")]
```
otherwise this test will not run with the testing suite

Reply:
Thanks for pointing it.